packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -106,15 +115,16 @@ describe(`request-queue`, function() {
       expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
       expect(callbacks.requestWillEnqueue.calledWith(sinon.match.has('request')))
           .to.be.true;
-
+      expect(performance.now() - startTime).to.be.above(800);
```

Comment:
@gauntface i have this `getDelayedPromise` which resolves after 800ms. This was added to make sure that the function does not proceed before the initialization promise is resolved